recipe_scrapers/hellofresh.py

```diff
@@ -11,7 +14,21 @@ def title(self):
         return self.schema.title()
 
     def total_time(self):
-        return self.schema.total_time()
+        script_tag = self.soup.find("script", {"id": "__NEXT_DATA__"})
+
+        if script_tag:
+            script_content = script_tag.string
+            total_time_match = re.search(r'"totalTime":"(PT\d+M)"', script_content)
+            prep_time_match = re.search(r'"prepTime":"(PT\d+M)"', script_content)
```

Comment:
I'm a little bit hesitant about this because the script block could be quite large.. but it does seem like a neat solution.

Out of interest: are these strings inside some JSON text within the script?  (basically, a `schema.org` entry but inside some JavaScript code?)

On balance I think we should probably merge this; I'll have a little bit more of a think about it.

Reply:
They seem to only appear in the __NEXT_DATA__ script but as you mentioned in the original issue the HTML is incredible hard to navigate.